components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/org/wso2/carbon/siddhi/tryit/ui/i18n/Resources.properties

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@prabathabey 
Thanks for pointing out. Fixed in PR: https://github.com/wso2/carbon-event-processing/pull/179